fvm/src/kernel/default.rs

```diff
@@ -57,37 +44,26 @@ const TEST_ACTOR_ALLOWED_TO_CALL_CREATE_ACTOR: ActorID = 98;
 pub struct DefaultKernel<C> {
     // Fields extracted from the message, except parameters, which have been
     // preloaded into the block registry.
-    caller: ActorID,
-    actor_id: ActorID,
-    method: MethodNum,
-    value_received: TokenAmount,
-    read_only: bool,
+    pub caller: ActorID,
+    pub actor_id: ActorID,
+    pub method: MethodNum,
+    pub value_received: TokenAmount,
+    pub read_only: bool,
```

Comment:
I'm starting to wonder how this will interact with testing. E.g., I want to support `FilecoinKernel<TestKernel<DefaultKernel>>`. Options include:

1. Using accessor methods here. IIRC, we never actually _change_ any of these fields (or the ones below), we just read them.
2. Require that all kernels "deref" to some `BaseKernel`.
3. Alternatively, require that all kernels expose some form of "invocation state"?

I think we can leave it alone for now, but I figured I'd comment on this in case you had any ideas.